try{
    const a=5;
    let b=8;
    const c=a+b;
    console.log(c);
}catch(error){
    console.log(error)
}